tests/pageobjects/side-nav.page.js

```diff
@@ -52,6 +52,8 @@ class SideNav extends Page {
 
 	get sidebarWrap() { return browser.element('.sidebar-wrap'); }
 
+	get sidebarItemMenu() { return browser.element('.sidebar-menu'); }
```

Comment:
I haven't found this element on the source code. Maybe you were trying to get `sidebar-item__menu` instead?

Reply:
Yes sorry